src/common/SurgeStorage.cpp

```diff
@@ -161,6 +161,16 @@ SurgeStorage::SurgeStorage()
    else
       datapath = std::string(homePath) + "/.local/share/Surge/";
```

Comment:
I think the linux way would be `/usr/share/surge` with lowercase, but this is fine for consistency with other platforms, too...

Reply:
Yeah lets leave it consistent for now and then revisit maybe later was my thinking.